ChaosMod/Effects/db/Player/PlayerRandomHud.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+	Effect By OnlyRealNubs
+*/
+
+#include "stdafx.h"
+
+static int comp = 0;

```

Comment:
Maybe just my coding OCD, but please rename to something self explanatory 😅 („compToHide“ and „availableComponents“ or something)

Reply:
a21ab69